tooling/noirc_abi/src/lib.rs

```diff
@@ -78,13 +78,15 @@ pub enum AbiVisibility {
     // Constants are not allowed in the ABI for main at the moment.
     // Constant,
     Private,
+    DataBus,
```

Comment:
This requires updates to the TS types in `noirc_abi_wasm` so that it can represent a type with databus visibility.